src/main/webapp/app/detail-overview-list/components/boolean-detail.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+import type { BooleanDetail } from 'app/detail-overview-list/detail.model';
+import { NoDataComponent } from 'app/shared/no-data-component';
+import { ArtemisSharedComponentModule } from 'app/shared/components/shared-component.module';
+
+@Component({
+    selector: 'jhi-boolean-detail',
+    template: `
+        <dd id="detail-value-{{ detail.title }}">
```

Comment:
I see that you just moved the code from the `detail-overview-list.ts` to the new component, but would it make sense to get rid of the `<dd> `tag? This way, we could use the new component more generically. Otherwise, it might cause errors if someone uses, for example, the` boolean-detail.component.ts` outside of a `<dl>` tag. While some browsers might attempt to render the page as best they can, this can lead to unpredictable results.

Reply:
Yes, this would be better, I just double checked it and we can move the <dd> tag as wrapper only in the `detail-overview-list.component.html` around the usage of the `jhiExerciseDetail` directive 👍 

Some elements have additional styling (e.g. `DetailType.ProgrammingProblemStatement`), but we can add this CSS in the `detail-overview-list.component.html`

Reply:
I will address this in the next directive PR